terraform/aws/components/nhais/variables.tf

```diff
@@ -78,7 +78,19 @@ variable "mq_broker_name" {
 variable "nhais_mongo_options" {
   type = string
   description = "Options for Mongo"
-  default = "replicaSet=rs0&readPreference=secondaryPreferred&retryWrites=false"
+  default = "replicaSet=rs0&readPreference=secondaryPreferred&retryWrites=false&tls=true"
```

Comment:
As TF may not allow default for variable to be based on other variable we may need to remove the default here and set it in global without the tls and with tls on envs where it is enabled